tsconfig.json

```diff
@@ -1,27 +1,18 @@
 {
   "compilerOptions": {
     "target": "esnext",
-    "module": "commonjs",
-    "lib": ["es6"],
+    "module": "es2015",
+    "lib": ["es6", "es2016", "es2018"],
     "jsx": "react-native",
-    "noEmit": true,
-    "isolatedModules": true,
     "strict": true,
     "noUnusedLocals": true,
     "noUnusedParameters": true,
     "noImplicitReturns": true,
     "noFallthroughCasesInSwitch": true,
+    "forceConsistentCasingInFileNames": true,
     "moduleResolution": "node",
     "allowSyntheticDefaultImports": true,
-    "esModuleInterop": true,
-    "baseUrl": "./src/",
-    "forceConsistentCasingInFileNames": true
+    "types": ["react", "react-native"]
```

Comment:
Curious @derrh -- what was the motivation behind this change?

Reply:
I misunderstood some errors I was experiencing when importing prim in another project. I thought that types from `hoist-non-react-statics` were causing issues and was trying to understand what impact this would have on prim's output. I've removed this entry.